web/frontend/src/app/component/subset-details/subset-details.component.html

```diff
@@ -44,19 +44,19 @@
                [lazy]="true" (onLazyLoad)="onLazyLoadData($event)" class="table table-striped table-bordered">
         <ng-template pTemplate="header">
           <tr>
-            <th class="w-10">Code</th>
-            <th class="w-10">Name</th>
-            <th class="w-40">Synonyms</th>
-            <th class="w-40">Definitions</th>
+            <th class="w-10">NCIt Concept Code</th>
+            <th class="w-10">NCIt Preferred Term</th>
+            <th class="w-40">NCIt Synonyms</th>
+            <th class="w-40">NCIt Definition</th>
           </tr>
         </ng-template>
         <ng-template pTemplate="body" let-value let-i="rowIndex">
           <tr>
-            <td class="w-10">{{ value.code }}</td>
-            <td class="w-10"><a [routerLink]="[urlBase, value.code ]" target='urlTarget'>{{ value.name }}</a></td>
+            <td class="w-10"><a [routerLink]="[urlBase, value.code ]" target='urlTarget'>{{ value.code }}<i class="fa fa-external-link" style="margin-left:0.2em;"></i></a></td>
+            <td class="w-10">{{ value.name }}</td>
             <td class="w-40">
               <span *ngFor="let syn of synonymSources[i] | keyvalue">
```

Comment:
Here we just want to show the "NCI" synonyms.  Take a look at the terms browser to see which ones it shows. I think we want the ones with source=NCI and those without a "source" value.